firmware_configurations/klipper/TechnoPhreak/BTT_SKR_v2/printer.cfg

```diff
@@ -0,0 +1,608 @@
+
+[mcu]
+##	MCU for X/Y/E steppers main MCU
+##	[X in X] - B Motor
+##	[Y in Y] - A Motor
+##	[E in E0] - Extruder
+##	Obtain definition by "ls -l /dev/serial/by-id/" then unplug to verify
+##--------------------------------------------------------------------
+serial: /dev/serial/by-id/usb-Klipper_stm32f407xx_00000-if00
+##--------------------------------------------------------------------
+
+[mcu z]
+##	MCU for Z steppers
+##	[Z in X] - Front Left
+##	[Z1 in Y] - Rear Left
+##	[Z2 in Z] - Rear Right
+##	[Z3 in E0]- Front Right
+##	Obtain definition by "ls -l /dev/serial/by-id/" then unplug to verify
+##--------------------------------------------------------------------
+serial: /dev/serial/by-id/usb-Klipper_stm32f407xx_00000-if00
+##--------------------------------------------------------------------
+
+[printer]
+kinematics: corexy
+max_velocity: 300  
+max_accel: 3000    			#Max 4000
+max_z_velocity: 15 			#Max 15 for 12V TMC Drivers, can increase for 24V
+max_z_accel: 350
+square_corner_velocity: 5.0
+
+#####################################################################
+# 	X/Y Stepper Settings
+#####################################################################
+
+[stepper_x]
+##	Connected to X on mcu_xye (B Motor)
+step_pin: PE2
+dir_pin: !PE1
+enable_pin: !PE3
+rotation_distance: 40
+microsteps: 16
+full_steps_per_rotation:200  #set to 400 for 0.9 degree stepper
+endstop_pin: ^PC1
+position_min: 0
+##--------------------------------------------------------------------
+
+##	Uncomment below for 250mm build
+#position_endstop: 250
+#position_max: 250
+
+##	Uncomment for 300mm build
+#position_endstop: 300
+#position_max: 300
+
+##	Uncomment for 350mm build
+#position_endstop: 350
+#position_max: 350
+
+##--------------------------------------------------------------------
+homing_speed: 25   #Max 100
+homing_retract_dist: 5
+homing_positive_dir: true
+
+##	Make sure to update below for your relevant driver (2208 or 2209)
+[tmc2209 stepper_x]
+uart_pin: PE0
+interpolate: True
+run_current: 0.8
+hold_current: 0.7
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+[stepper_y]
+##	Connected to Y on mcu_xye (A Motor)
+step_pin: PD5
+dir_pin: !PD4
+enable_pin: !PD6
+rotation_distance: 40
+microsteps: 16
+full_steps_per_rotation:200  #set to 400 for 0.9 degree stepper
+endstop_pin: ^PC3
+position_min: 0
+##--------------------------------------------------------------------
+
+##	Uncomment for 250mm build
+#position_endstop: 250
+#position_max: 250
+
+##	Uncomment for 300mm build
+#position_endstop: 300
+#position_max: 300
+
+##	Uncomment for 350mm build
+#position_endstop: 350
+#position_max: 350
+
+##--------------------------------------------------------------------
+homing_speed: 25  #Max 100
+homing_retract_dist: 5
+homing_positive_dir: true
+
+##	Make sure to update below for your relevant driver (2208 or 2209)
+[tmc2209 stepper_y]
+uart_pin: PD3
+interpolate: True

```

Comment:
Please remove hold_current, set interpolate to false and microsteps to 32 for all steppers to align with the new klipper recommendations.

Reply:
Should be good now